include/picongpu/particles/atomicPhysics/kernel/FillLocalRateCache_BoundBound.kernel

```diff
@@ -154,7 +154,6 @@ namespace picongpu::particles::atomicPhysics::kernel
 
             auto forEachAtomicStateAndBin
                 = pmacc::lockstep::makeForEach<T_numberAtomicStates * T_numberBins, T_Worker>(worker);
```

Comment:
Sorry for being pedantic but the same applies to this one. It should also move into the `if constexpr`.

Reply:
Yes, you are right possible to change in a separate PR. I only react on broken tests in the CI I have not done a code refactoring.

Reply:
Okay.